### PR TITLE
Add docs for responsive images

### PR DESCRIPTION
**Pull request by alexjoverm**
@aarongarciah let's iterate over this PR in order to define the docs. I'll need some guidance since I haven't use the `srcset` attributes that much.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
-  <div>
-    <v-lazy-image
-      src="https://cdn-images-1.medium.com/max/1600/1*xjGrvQSXvj72W4zD6IWzfg.jpeg"
-      src-placeholder="https://cdn-images-1.medium.com/max/80/1*xjGrvQSXvj72W4zD6IWzfg.jpeg"
-      ></v-lazy-image>
-  </div>
+  <v-lazy-image
+    src="https://cdn-images-1.medium.com/max/1600/1*xjGrvQSXvj72W4zD6IWzfg.jpeg"
+    src-placeholder="https://cdn-images-1.medium.com/max/80/1*xjGrvQSXvj72W4zD6IWzfg.jpeg"
+    ></v-lazy-image>
```
Comment (aarongarciah) [inline README.md]: How about writing the component like

File: README.md [hunk 2]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
-## Progressive loading
+## Responsive Images
+
+Using the `srcset` prop in combinations with the `sizes` attribute you can set images for different sizes:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 1x, image_2x.jpg 2x"
+    ></v-lazy-image>
```
Comment (aarongarciah) [inline README.md]: How about writing the components without ending tag?
```
<v-lazy-image srcset="image.jpg 1x, image_2x.jpg 2x" />
```

File: README.md [hunk 3]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
 
-## Progressive loading
+## Responsive Images
+
+Using the `srcset` prop in combinations with the `sizes` attribute you can set images for different sizes:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 1x, image_2x.jpg 2x"
```
Comment (aarongarciah) [inline README.md]: When using the `srcset` attribute is recommended to use also `src` in order to keep the browsers that doesn't support the `srcset` attribute.

File: README.md [hunk 4]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
 
 That image will be loaded as soon as the image enters the viewport.
 
-## Progressive loading
+## Responsive Images
+
+Using the `srcset` prop in combinations with the `sizes` attribute you can set images for different sizes:
```
Comment (aarongarciah) [inline README.md]: I would rewrite this sentence as

File: README.md [hunk 5]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
+```
+
+It's possible to use it with the `sizes` attributes as well:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 400w, image.jpg 800w"
+    sizes="(max-width: 400w) 400w, 100vw"
+    ></v-lazy-image>
```
Comment (aarongarciah) [inline README.md]:
```

Comment (aarongarciah) [inline README.md]:
```html
<v-lazy-image
	srcset="image-320w.jpg 320w, image-480w.jpg 480w"
	sizes="(max-width: 320px) 280px, 440px"
	src="image-480w.jpg"
/>
```

File: README.md [hunk 6]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 1x, image_2x.jpg 2x"
+    ></v-lazy-image>
+</template>
+```
+
+It's possible to use it with the `sizes` attributes as well:
```
Comment (aarongarciah) [inline README.md]: You can combine the `srcset` prop with the `sizes` attribute

File: README.md (hunk 2 above)
Comment (alexjoverm) [reply README.md]: Yeah, that's better!

File: README.md [hunk 7]
```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
...
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 400w, image.jpg 800w"
+    sizes="(max-width: 400w) 400w, 100vw"
+    ></v-lazy-image>
+</template>
+```
+
+## Progressive Loading
```
Comment (aarongarciah) [inline README.md]: Maybe is good to place the *Progressive Loading* before the *Responsive Images* and then in the **

**Comment by aarongarciah** [comment]
I've forked your demo to make the responsive images work. https://codesandbox.io/s/k2kp64qkq7

**Comment by alexjoverm** [comment]
Thanks @aarongarciah!! All changes are applied and I went ahead and release https://github.com/alexjoverm/v-lazy-image/releases/tag/1.1.0. Feel free to open a new issue if you find something is still missing.

**Comment by aarongarciah** [comment]
Looks good to me 😄

**Pull request merged**